### DISCUSSION TITLE
plugins or integration with workflow tools

### DISCUSSION THREAD

**Discussion by lnielsen** [inveniosoftware/product-rdm#138] [Ideas]
plugins or integration with workflow tools like Jupyter notebooks, R, GitHub Zeppelin Notebook, and tools @ CERN that support + reproducibility + citation/credit like Asclepias/Reana.

**Comment by tmorrell** [comment]
I've got binder integration set up in CaltechDATA, but it's a manual operation to add the badge to the record. Some sort of integrated  validation workflow would be cool https://data.caltech.edu/records/1250

Once the the data and file model is set, I'd be happy to update https://github.com/jupyter/repo2docker/blob/master/repo2docker/contentproviders/zenodo.py to invenioRDM

[a comment by an automated account is not shown]
